starknet/src/voting_strategies/erc20_votes.cairo

```diff
@@ -0,0 +1,42 @@
+use sx::types::user_address::UserAddressTrait;
+#[starknet::contract]
+mod ERC20VotesVotingStrategy {
+    use sx::interfaces::IVotingStrategy;
+    use sx::types::{UserAddress, UserAddressTrait};
+    use starknet::ContractAddress;
+    use serde::Serde;
+    use array::{ArrayTrait, SpanTrait};
+    use openzeppelin::governance::utils::interfaces::votes::{
+        IVotes, IVotesDispatcher, IVotesDispatcherTrait
+    };
+    use traits::Into;
+
+    use option::OptionTrait;
+
+    #[storage]
+    struct Storage {}
+
+    #[external(v0)]
+    impl ERC20VotesVotingStrategy of IVotingStrategy<ContractState> {
+        fn get_voting_power(
+            self: @ContractState,
+            timestamp: u32,
+            voter: UserAddress,
+            params: Array<felt252>,
+            user_params: Array<felt252>,
+        ) -> u256 {
+            // Cast voter address to a Starknet address
+            // Will revert if the address is not a Starknet address
+            let voter = voter.to_starknet_address();
+
+            // Get the ERC20 contract address from the params array
+            let mut params_span = params.span();
```

Comment:
can we just shadow the var in these instances? like `let mut params = params`

Reply:
091aaf360dfa3efd269cd3882a4a88d06058747c